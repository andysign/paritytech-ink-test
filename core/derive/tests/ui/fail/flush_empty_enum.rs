use ink_core_derive::Flush;

#[derive(Flush, Debug, PartialEq, Eq)]
enum EmptyEnum {}

fn main() {}
