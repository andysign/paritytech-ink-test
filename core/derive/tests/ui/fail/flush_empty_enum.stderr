error: proc-macro derive panicked
 --> $DIR/flush_empty_enum.rs:3:10
  |
3 | #[derive(Flush, Debug, PartialEq, Eq)]
  |          ^^^^^
  |
  = help: message: deriving Flush for empty enum is invalid
